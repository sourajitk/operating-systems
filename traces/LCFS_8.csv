1,0,0
2,0,0
3,0,0
4,0,100
5,101,5
6,101,5
7,101,5
8,101,200
9,500,100
10,500,0
11,500,0
12,500,0
13,800,200
14,800,0
15,800,0
16,800,0
17,1000,300
18,1000,0
19,1000,0
20,1000,0
