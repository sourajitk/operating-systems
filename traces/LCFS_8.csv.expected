1,0
2,0
3,0
4,100
5,106
6,316
7,311
8,306
9,600
10,600
11,600
12,600
13,1000
14,1000
15,1000
16,1000
17,1300
18,1300
19,1300
20,1300
